datadog_checks_dev/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG - Datadog Checks Dev
 
+## 17.4.0 / 2022-11-23
+
+* [Added] Add a dummy --hide-memray-summary option to the pytest plugin. See [#13358](https://github.com/DataDog/integrations-core/pull/13358).
```

Comment:
you might want to remove this one too since you removed the --memray line

Reply:
changed labels, wdyt?

Reply:
looks good to me!